blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfig.java

```diff
@@ -44,13 +45,13 @@ private String createConfig() {
                     .key("security")
                     .object()
                         .key("enabled").value(jenkins.isUseSecurity())
-                        .key("authorizationStrategy")
-                        .object()
+                        .key("authorizationStrategy").object()
                             .key("allowAnonymousRead").value(allowAnonymousRead)
+                            .endObject()
+                        .key("enableJWT").value(FEATURE_JWT)
```

Comment:
Does this property accessible as security.jenkinsConfig.enableJwt? Is it not more of BO specific feature so maybe it should be security.enableJwt? or maybe just top level enableJwt?